modules/rls/notify.c

```diff
@@ -119,12 +119,11 @@ int send_full_notify(subs_t* subs, xmlNodePtr service_node, int version, str* rl
 		LM_ERR("in sql query\n");
 		goto error;
 	}
-	if(result== NULL)
-		goto error;
 	*/
 
 	// build result instead of query in DB since initial query result is guaranteed to be empty
-    result = build_db_result(service_node, n_result_cols);
+    if ((result = build_db_result(service_node, n_result_cols)) == 0)
```

Comment:
Fix your tabbing to be consistent with existing coding style.  I see the same issue throughout this commit.